src/main/java/net/dv8tion/jda/core/managers/impl/AudioManagerImpl.java

```diff
@@ -65,8 +66,8 @@
 
     public AudioManagerImpl(Guild guild)
     {
-        this.guild = guild;
-        this.api = (JDAImpl) guild.getJDA();
+        this.guild = (GuildImpl) guild;
```

Comment:
Thanks for clarifying, although that makes partially sense to me: can't one simply mess with the methods and fields of GuildImpl?